settings.py

```diff
@@ -22,3 +22,18 @@
 to be at least greater than FLOW_STATS and ideally it slightly greater than
 whichever longest network convergence FlowMods operations that your network has.
 """
+
```

Comment:
David, if you could also add more comments here regarding that it's per napp and per dpid and how to disable it that'd be informative for network engineers.

Reply:
Added in the comments here.

Reply:
Cool. Can you also explain how to disable if they ever need to? We don't expect they'll need to it, after all the pacer is key to overall stabilize things, but if they're debugging or whatever that's good to know, we used to have it on the parts you'll replace https://github.com/kytos-ng/mef_eline/blob/master/settings.py#L36